### ISSUE TITLE
2fa-installer not taking into consideration password policy

### ISSUE THREAD

**Issue by stasou** [Zimbra-Community/zimbra-foss-2fa#7]
Hello, I have a single server installation and I run the installer script, as there was no apparent need for the manual process.

All went well till the part that the ldap-2fa (bind) account was being created.

Then I got this error:

prov> ca sa-ldap-2fa@example.com some_weird_looking_pasword
ERROR: account.INVALID_PASSWORD (invalid password: not enough numeric characters)

Not a big deal, actually, I manually got over this part in less than a minute, but perhaps policy should be taken into account when creating the account password, OR create a password that contains a little bit of everything.

A hint would be to run

`zmprov gc default | grep zimbraPasswordMin | grep -v " 0"`

and append a little bit of what comes out

Great work though, all together, I would like to point this out. GREAT WORK.

Kind regards,
Stavros

**Comment by barrydegraaff** [comment]
Did you set a custom policy? Or did this happen with a default set-up?

The password is a 24 character random:

SERVICEACCT_PWD=$(< /dev/urandom tr -dc _A-Z-a-z-0-9 | head -c${1:-24};echo;)

Which in theory could have zero numbers in it...

**Comment by stasou** [comment]
Yes, there is a custom policy, enforcing at least 1 of numeric, lowercase and symbol.

You do know Murphy's law, don't you? Of course out of 24 characters, not one would be something required :) . Just a bit of poor luck.

However, policy can be skipped when setting a password for an existed account.

So, I solved this by appending the missing part and then just removing it:
`zmprov ca sa-ldap-2fa@example.com some_weird_looking_pasword1`
`zmprov sp sa-ldap-2fa@example.com some_weird_looking_pasword`

I just continued testing by the way, everything seems great. There is one other suggestion, but I will post in another issue.
